### PR TITLE
fix combat, add version, remove api docs

### PR DESCRIPTION
**Pull request by dni**
i need some feedback here, i broke some compatibility with lnurldevice because i changed the query params inside the LNURL to make them more descriptive.
now i introduced /v2/ of that endpoint and keep /v1/ like it was before, for backwards compatibility. 

can i do it like that? 

i think its the cleanest solution else i would need to introduce more optional arguments for that endpoint which would make the code very confusing.

added update warning
screenshot-1678962886

### CODE REVIEW

**Comment by callebtc** [comment]
"because i changed the query params inside the LNURL to make them more descriptive" Where did you do this?

File: lnurl.py [hunk 1]
```diff
@@ -64,13 +63,42 @@ def xor_decrypt(key, blob):
...
+):
+    return await lnurl_params(request, device_id, p, atm, pin, amount, duration)
+
+
+async def lnurl_params(
+    request: Request,
+    device_id: str,
+    p: str,
+    atm: str,
+    pin: str,
```
Comment (dni) [inline lnurl.py:99]: here, changed to pin - amount - duration instead auf gpio - profit - amount

File: lnurl.py [hunk 2]
```diff
@@ -64,13 +63,42 @@ def xor_decrypt(key, blob):
...
+    name="lnurldevice.lnurl_v2_params",
+)
+async def lnurl_v2_params(
     request: Request,
     device_id: str = Query(None),
     p: str = Query(None),
     atm: str = Query(None),
     pin: str = Query(None),
     amount: str = Query(None),
     duration: str = Query(None),
```
Comment (dni) [inline lnurl.py:89]: here, changed to pin - amount - duration instead auf gpio - profit - amount

Comment (dni) [reply lnurl.py:89]: we use amount all the time for invoices, i think using amount for the duration how long the switch is triggered is misleading.

Comment (AxelHamburch) [reply lnurl.py:89]: The option to change the time and amount afterwards really useful. This is very welcome. It should be compatible with the already install bitcoinSwitch. There are quite a few already. In so far this is a good solution. 

The display of the extension version is really useful. The warning would be perhaps additionally also with the selection and option to install still appropriate. Then you know immediately what you have to adjust and can warn the user if necessary.

**Comment by AxelHamburch** [comment]
The option to change the time and amount afterwards really useful. This is very welcome. It should be compatible with the already install bitcoinSwitch. There are quite a few already. In so far this is a good solution. 

The display of the extension version is really useful. The warning would be perhaps additionally also with the selection and option to install still appropriate. Then you know immediately what you have to adjust and can warn the user if necessary.

**Pull request merged**